src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -52,30 +51,13 @@ const DataViewTable: React.FC = () => {
     filtersReady: !filtersLoading,
   })
 
-  const [currentSortColumn, setCurrentSortColumn] = useState<SortColumn>(null)
```

Comment:
Can get rid of quite a lot as a result of not showing the currently sorted column in the filter bar 👍🏼  (Most of the removed content)